indigo-frontend/src/app/pages/template/template-list/template-list.component.ts

```diff
@@ -0,0 +1,109 @@
+import { Component, inject, Input, OnDestroy } from '@angular/core';
```

Comment:
Please get rid of project-related stuff like opening project creation dialog and so on.